ui/src/app/components/resource-card-template.tsx

```diff
@@ -54,28 +64,60 @@ const styles = reactStyles({
   }
 });
 
-export interface Action {
+interface Action {
   icon: string;
   displayName: string;
   onClick: () => void;
   disabled: boolean;
   hoverText?: string;
 }
 
+const ResourceActionsMenu = (props: {actions: Action[]}) => {
```

Comment:
moved up from below in the file, and made into a component.  I'll reuse this for RW-5562.